src/utils/ConfigHelper.ts

```diff
@@ -129,6 +129,33 @@ export const configHelperNetworks: ConfigHelperConfig[] = [
     providerUri: 'https://provider.bsc.oceanprotocol.com',
     subgraphUri: 'https://subgraph.bsc.oceanprotocol.com',
     explorerUri: 'https://bscscan.com/'
+  },
+  {
+    ...configHelperNetworksBase,
+    networkId: 44787,
+    network: 'celoalfajores',
+    nodeUri: 'https://alfajores-forno.celo-testnet.org',
+    providerUri: 'https://provider.celoalfajores.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.celoalfajores.oceanprotocol.com',
+    explorerUri: 'https://alfajores-blockscout.celo-testnet.org'
+  },
+  {
+    ...configHelperNetworksBase,
+    networkId: 246,
+    network: 'energyweb',
+    nodeUri: 'https://rpc.energyweb.org',
+    providerUri: 'https://provider.energyweb.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.energyweb.oceanprotocol.com',
+    explorerUri: 'https://explorer.energyweb.org'
+  },
+  {
+    ...configHelperNetworksBase,
+    networkId: 1285,
+    network: 'moonriver',
+    nodeUri: 'https://moonriver.api.onfinality.io/public',
```

Comment:
That's a 3rd party service. Does it work without providing an API key or does market have to inject that in end just like with Infura? If so which account and plan are we using for this>

And from their docs:

> Our API Service is currently in beta. We've got big plans for it, but please treat it as such and avoid running production applications on it.

https://support.onfinality.io/hc/en-us/articles/900004222183-API-Service-Getting-Started

Reply:
it's one of the official RPCs:  https://docs.moonbeam.network/builders/get-started/moonriver/